x/leverage/fixtures/token.go

```diff
@@ -0,0 +1,31 @@
+package fixtures
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/umee-network/umee/v2/x/leverage/types"
+)
+
+// Token returns a valid token
+func Token(base, symbol, reserveFactor string) types.Token {
```

Comment:
How about `types.ExampleToken()` instead of `fixtures.Token(base,symbol,reserveFactor)`? And have it return `uumee`/`UMEE` with sensible values.

That way
- We don't need a new folder/package
- We cover most of the repeated code in the test cases
- Reserve factor doesn't get special treatment (since any of the parameters can be customized after initializing)

Reply:
1. We should not mix test domain with API domain. This data is part of the test domain, hence should be in a separate package
2. the package is called `fixtures`, so we don't need `Example` prefix.
3. I used `base, symbol, reserveFactor` because it was used like that in other test files (see tests).

Reply:
I can check if we can remove `reserveFactor` argument.